WalletWasabi.Fluent/Converters/WalletIconConverter.cs

```diff
@@ -8,9 +8,9 @@ namespace WalletWasabi.Fluent.Converters;
 public static class WalletIconConverter
 {
 	public static readonly IValueConverter WalletTypeToImage =
-		new FuncValueConverter<WalletType, Bitmap>(GetBitmap);
+		new FuncValueConverter<WalletType, Bitmap?>(GetBitmap);
 
-	private static Bitmap GetBitmap(WalletType type)
+	private static Bitmap? GetBitmap(WalletType type)
```

Comment:
Fix `AssetHelpers.GetBitmapAsset()` to not allow null and marking these as nullable is not required.

Reply:
done.